src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.spec.ts

```diff
@@ -4,8 +4,8 @@ import { MockStore, provideMockStore } from '@ngrx/store/testing';
 
 import { CreateCustomerComponent } from './create-customer';
 import { CustomerState, CreateCustomer } from 'src/app/modules/customer-management/store';
-import * as models from 'src/app/modules/shared/models/index';
 import { LoadCustomers } from './../../../../store/customer-management.actions';
+import * as models from 'src/app/modules/shared/models/index';
```

Comment:
import should be
**import { Customer } from 'src/app/modules/shared/models';**

Reply:
done